sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1774,6 +1773,21 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  object IntervalStyle extends Enumeration {
+    type IntervalStyle = Value
+    val SQL_STANDARD, ISO_8601, MULTI_UNITS = Value
+  }
+
+  val INTERVAL_STYLE = buildConf("spark.sql.intervalOutputStyle")
```

Comment:
fyi: we might need to move this config into `spark.sql.dialect.xxx` along https://github.com/apache/spark/pull/26444

Reply:
I don't know,  but the behavior of this config is beyond the meaning of a `dialect`.